src/Dhall/Import.hs

```diff
@@ -176,11 +178,11 @@ import qualified Dhall.Parser
 import qualified Dhall.Context
 import qualified Dhall.TypeCheck
 import qualified Filesystem
-import qualified Filesystem.Path.CurrentOS
 import qualified Network.HTTP.Client              as HTTP
 import qualified Network.HTTP.Client.TLS          as HTTP
 import qualified Filesystem.Path.CurrentOS        as Filesystem
 import qualified System.Environment
+import qualified System.Directory
```

Comment:
What's the reason for the `System.Directory` import?  It seems like it's not used

Reply:
Oh right, I played around with `makeAbsolute` and forgot to remove it once I saw that it leads nowhere.